tests/spec/test_json_schema_spec.py

```diff
@@ -20,7 +20,7 @@
 
 VALID_SCHEMA_DIR = "tests/spec/valid"
 INVALID_SCHEMA_DIR = "tests/spec/invalid"
-RECAP_SPEC_JSON_HTTP = "https://recap.build/specs/type/0.2.0.json"
+RECAP_SPEC_JSON_HTTP = "https://recap.build/specs/type/0.3.0.json"
```

Comment:
I see that this is why the tests are failing, the website change hasn't landed yet. 👍

Reply:
Yep! PR for that is here:

https://github.com/recap-build/recap-website/pull/13